lib/registries/addon/overview/template.hbs

```diff
@@ -64,8 +64,8 @@
                         @models={{array this.registration.id}} @icon='home' @label={{t 'registries.overview.title'}} />
                     {{#if this.registrationFilesPageEnabled}}
                         <leftNav.link data-analytics-name='Files' 
-                            @route='registries.overview.files'
-                            @models={{array this.registration.id}}
+                            @route='registries.overview.files.provider'
+                            @models={{array this.registration.id 'osfstorage'}}
```

Comment:
I don't think changing this is necessary, since `registries.overview.files.index` will forward users to the `providers` subroute in the beforeModel hook. I checked and the link will also not be active if you are in the `outputs` subroute

Reply:
Innnteresting. I wonder what the specifics were that we had to fix it for the project files. Oh, well, not a big deal.